src/DBDiffer/SQL/DBCreator.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace jach\DBdiffer\SQL;
+
+class DBCreator
```

Comment:
Den her klasse laver to forskellige ting. Loader statements fra filer og smider i en DB. Jeg ville splitte loading a SQL fra filer over i en klasse for sig som leverer et array af SQL statements tilbage, som så kan smides videre til differen.